src/main/java/in/twizmwaz/cardinal/module/modules/startTimer/StartTimer.java

```diff
@@ -74,6 +89,22 @@ public void run() {
                     }
                 }
             }
+            for (Player player : Bukkit.getOnlinePlayers()) {
+                if (!TeamUtils.getTeamByPlayer(player).isObserver()) {
+                    if (time == 3) {
```

Comment:
@CaptainElliott @twizmwazin Modulus in Java is `%`. Not sure if `%%` has a different function.